planning/behavior_path_planner/include/behavior_path_planner/scene_module/avoidance/avoidance_module.hpp

```diff
@@ -325,18 +325,35 @@ class AvoidanceModule : public SceneModuleInterface
    * 3. merge raw shirt lines.
    * 4. trim unnecessary shift lines.
    */
-  AvoidLineArray applyPreProcessToRawShiftLines(
+  AvoidLineArray applyPreProcess(
     AvoidLineArray & current_raw_shift_points, DebugData & debug) const;
 
-  AvoidLineArray getFillGapShiftLines(const AvoidLineArray & shift_lines) const;
+  /*
+   * @brief fill gap among shift lines.
+   * @param original shift lines.
+   * @param debug data.
+   * @return processed shift lines.
+   */
+  AvoidLineArray applyFillGapProcess(const AvoidLineArray & shift_lines, DebugData & debug) const;
 
   /*
    * @brief merge negative & positive shift lines.
    * @param original shift lines.
    * @param debug data.
    * @return processed shift lines.
    */
-  AvoidLineArray mergeShiftLines(const AvoidLineArray & raw_shift_lines, DebugData & debug) const;
+  AvoidLineArray applyMergeProcess(const AvoidLineArray & shift_lines, DebugData & debug) const;
+
+  /*
+   * @brief add return shift line from ego position.
+   * @param shift lines which the return shift is added.
+   * Pick up the last shift point, which is the most farthest from ego, from the current candidate
+   * avoidance points and registered points in the shifter. If the last shift length of the point is
+   * non-zero, add a return-shift to center line from the point. If there is no shift point in
+   * candidate avoidance points nor registered points, and base_shift > 0, add a return-shift to
+   * center line from ego.
+   */
+  AvoidLineArray addReturnShiftLine(const AvoidLineArray & shift_lines, DebugData & debug) const;
```

Comment:
@satoshi-ota
Explanation is very clear! sasuga!
minor proposal you can ignore as always :smile
Return → RetunToCenter, hmm too long...?

Reply:
description below is clear enough so don't need to change function name 
`return-shift to center line from ego.`
sorry

Reply:
> minor proposal you can ignore as always :smile
:kusa: